### PR TITLE
Removed the non-functional checkbox in default state

### PR DESCRIPTION
**Pull request by yellowwoods12**
In reference to issue: #1528 
Here's how the popup looks when no container is open (removed the non-functional checkbox)

Screenshot from 2019-10-19 22-59-56

### CODE REVIEW

**Comment by yellowwoods12** [comment]
Hi @maxxcrawford and @groovecoder, Can you please review this and suggest any changes if required?

File: src/js/popup.js [hunk 1]
```diff
@@ -615,8 +615,14 @@ Logic.registerPanel(P_CONTAINERS_LIST, {
...
       const currentContainer = document.getElementById("current-container");
+      const DefaultText =document.getElementById("default-text");
+      const DefaultBox =document.getElementById("container-page-assigned");
       currentContainer.innerText = identity.name;
-
+      if (currentContainer.innerText === "Default"){
+        DefaultBox.style.display="none";
+        DefaultText.style.display="none";
+        currentContainer.style.display="none";
+      }
```
Comment (groovecoder) [inline src/js/popup.js]: you could actually do the same effective thing with a single line:
```js
if (identity.name === "Default") {
  document.getElementById("container-page-assigned-label").style.display = "none";
}
```

File: src/popup.html [hunk 2]
```diff
@@ -101,7 +100,7 @@ <h3>Current Tab</h3>
       <div id="current-page"></div>
       <label for="container-page-assigned">
```
Comment (groovecoder) [inline src/popup.html]: If you add `id="container-page-assigned-label"` attribute here (and don't need to add `id="default-text"` below, you can hide the whole block of elements with the single line mentioned above.

**Comment by yellowwoods12** [comment]
Hi @groovecoder, I've made the necessary changes as per your review. kindly have a look into it and let me know.

**Comment by maxxcrawford** [comment]
Closing this PR as it applies to pre-7.0 [UI flow](https://github.com/mozilla/multi-account-containers/releases/tag/7.0.0).

**Pull request closed without merging**